repository/src/main/java/org/apache/atlas/repository/graph/GraphBackedSearchIndexer.java

```diff
@@ -415,9 +415,9 @@ private void initialize(AtlasGraph graph) throws RepositoryException, IndexExcep
             createVertexCentricIndex(management, CLASSIFICATION_LABEL, AtlasEdgeDirection.BOTH, Arrays.asList(CLASSIFICATION_EDGE_NAME_PROPERTY_KEY, CLASSIFICATION_EDGE_IS_PROPAGATED_PROPERTY_KEY));
 
             // create edge indexes
-            createEdgeIndex(management, RELATIONSHIP_GUID_PROPERTY_KEY, String.class, SINGLE, true);
-            createEdgeIndex(management, EDGE_ID_IN_IMPORT_KEY, String.class, SINGLE, true);
-            createEdgeIndex(management, ATTRIBUTE_INDEX_PROPERTY_KEY, Integer.class, SINGLE, true);
+            createEdgeIndex(management, RELATIONSHIP_GUID_PROPERTY_KEY, String.class, SINGLE, true, false);
```

Comment:
As these were not keywords earlier, do we need to migrate the existing ones. AFAIK this change won't be present in older data

Reply:
Old indexing will take it as TEXT rather than the keyword

Reply:
We really do not care about existing mappings on the edge_index for now.
We only focussed on enabling it for the relationshipAttributes we will be adding going forward.

Reply:
Yup got it